Astro Release Notes for April 21 + 28 (#644)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.4.0 ([Release notes](cli-release-notes.md))

## April 28, 2022

### New AWS Node Instance Types Available

> To widen our support for complex use cases at scale, we've grown the types of AWS node instances that are supported on Astro. You can now create Clusters with:

Author: ```suggestion
To widen our support for various use cases and levels of scale, we've expanded the types of AWS node instances that are supported on Astro. You can now create Clusters with:
```

- [General Purpose M6i instances](https://aws.amazon.com/ec2/instance-types/m6i/)
- [Compute Optimized C6i instances](https://aws.amazon.com/ec2/instance-types/c6i/)
- [Memory Optimized R6i instances](https://aws.amazon.com/ec2/instance-types/r6i/)

For a full list of node instance types that are supported on Astro, see [AWS Resource Reference](resource-reference-aws.md#node-instance-type). To modify an existing Astro Cluster to use any of these instance types, see [Modify a Cluster](modify-cluster.md).

### Additional Improvements

> - Improve the error message that renders in the Cloud UI if you try to create a worker that is too large for the Deployment's node instance type to support. This error message now specifies a clear call to action

Author: Weird thing, but we don't do periods in these lists ugh it also bugs me - we may change style guide
